_layouts/default.html

```diff
@@ -4,10 +4,12 @@
   <body class="grid-block vertical body page-landing">
     {% include icons.html %}
     {% include header.html %}
-    <main class="grid-block grid-container-lg main-card">
-      {% include sidebar.html %}
-      <div class="grid-block vertical lunr-hidden" id="results"></div>
-      <div class="grid-block vertical lunr-active" id="content">{{ content }}</div>
+    <main class="grid-block grid-container-lg">
```

Comment:
Instead of putting the .grid-container-lg class on the same element that gets the padding, let's put it on the child of this element. That way we get the width we want for the content without having to modify box-sizing.

Also, since the child of this element is the .main-card, let's give it the <main> tag instead. So the final markup would look like this:

```
<div class="grid-block padding-left-sm padding-right-sm">
  <main class="grid-block main-card grid-container-lg">
```